### PR TITLE
Cleanup request data in local thread

### LINKED ISSUE #16843: Cleanup request thread data
WebContainer does not clean up the request local thread data correctly.

### PR DESCRIPTION
**Pull request by pmd1nh**
https://github.com/OpenLiberty/open-liberty/issues/16843

### CODE REVIEW

**Comment by pmd1nh** [comment]
#build     #spawn.fullfat.buckets=com.ibm.ws.webcontainer_fat,com.ibm.ws.webcontainer.servlet.3.1_fat,com.ibm.ws.webcontainer.servlet.4.0_fat,com.ibm.ws.webcontainer_fat_lWAS,com.ibm.ws.webcontainer_fat_servlet31_webcontainerlWAS,com.ibm.ws.webcontainer.servlet.4.0_fat_toleration.3.0,com.ibm.ws.webcontainer.servlet.4.0_fat_toleration.3.0.lWAS

**Comment by pmd1nh** [comment]
#run-libby-bot

File: dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/srt/SRTServletRequestThreadData.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2016 IBM Corporation and others.
+ * Copyright (c) 2021 IBM Corporation and others.
```
Comment (jhanders34) [inline dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/srt/SRTServletRequestThreadData.java]: This should be 2016,2021

**Review by jhanders34 — APPROVED** [review]

**Review by volosied — APPROVED** [review]

**Comment by pmd1nh** [comment]
#run-libby-bot

**Pull request merged**